pyproject.toml

```diff
@@ -29,8 +41,8 @@ black = true
 close-quotes-on-newline = true
 exclude = ["build"]
 recursive = true
-wrap-descriptions = 79
-wrap-summaries = 79
+wrap-descriptions = 88
+wrap-summaries = 88
```

Comment:
I request that we revert this change:  there are good reasons for 79 columns for English text (just as there are good reasons for ~88 columns for code), and there is no reason to impose the same limit on the former just because it is our choice for the latter.

(Also, doing so is likely to greatly reduce the number of lines, indeed the number of files, changed in this PR....)

Reply:
Fixed.